### ISSUE TITLE
Copy path to clipboard

### ISSUE THREAD

**Issue by AndrewM-** [mikekov/ExifPro#1]
I use linked files a lot and getting a file address out of ExifPro is really difficult.  When I drag an image out of a web browser into NotePad++, I get a file address.  I would love to be able to do this from ExifPro or even just right click on an image and copy the file address.

**Comment by utak3r** [comment]
You mean copying the full path to the file (including the filename itself)? I'm also using this sometimes in other apps.

**Comment by AndrewM-** [comment]
Yes that is exactly correct.  I replied via email back in January and it is disappointing to see the response did not make it to this issue list.  Thanks for asking.

**Comment by mikekov** [comment]
Marked as "enhancement". I didn't realize I need to tag it. Still learning github.

**Comment by AndrewM-** [comment]
I am still learning github as well.  I built an image information system in ms-access which manages about 250 K images of plants and animals but I need to be able to drag in file addresses for local files.  Currently, I use ms-access to generate a hand coded web page and drag in the photos from there.  ExifPro is unique in terms of speed and also in being able to size images and thumbnails for viewing.  Sometimes I have an ultra-hires monitor and can't see normal sized thumbnails well. ExifPro is a major part of my workflow so I would really appreciate the feature.

**Comment by mikekov** [comment]
I just wanted to set expectations: I'm happy to accept feature requests, but as you can see from project's history commits are infrequent. I'm no longer actively working on ExifPro, but of course pull requests are welcome.

**Comment by AndrewM-** [comment]
I am very happy that there is some activity.  My own development activity is heading in a different direction and it is unlikely that I will develop the skills to copy a file name to the clipboard.  It is frustrating as I can seen what I want presented as text in the UI, but it is not selectable.
